tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -88,7 +86,6 @@
  * The @MultipartConfig annotation enables getting request parameters from a POST request
  * that has "multipart/form-data" as its content-type, as needed for WHAT_USER_LIST.
  */
-@WebServlet
 @MultipartConfig
```

Comment:
I had thought that @WebServlet might be required along with @MultipartConfig, but @WebServlet turns out to be problematic as well as unneeded.

If we have simply @WebServlet, we get: "[ERROR ] SRVE9989E: A @WebServlet annotation on the class org.unicode.cldr.web.SurveyAjax has neither a value nor a urlPatterns attribute. Either value or urlPatterns must be specified."

If we have @WebServlet("/SurveyAjax"), we get: "[ERROR ] SRVE9016E: Unable to insert mapping [/SurveyAjax] for servlet named [org.unicode.cldr.web.SurveyAjax]. The URL pattern is already defined for servlet named [SurveyAjax]."

I guess that involves what is "already defined" in web.xml.

Reply:
I kind of had thought we would add new parts under teh new 'api' package and move away from new items under SurveyAjax , but this sounds fine